core/src/main/java/io/grpc/ManagedChannel.java

```diff
@@ -115,4 +123,10 @@ public void notifyWhenStateChanged(ConnectivityState source, Runnable callback)
    */
   @ExperimentalApi
   public void resetConnectBackoff() {}
+
+  @Internal
+  @Override
+  public InternalLogId getLogId() {
+    return InternalLogId.allocate(getClass().getName());
```

Comment:
The logId should be a final field.

Also, there doesn't seem to be a case where a subclass needs to override this method. Can we make this method final? All current overrides could be simply removed.

Reply:
Done.